build.rs

```diff
@@ -60,6 +66,37 @@ fn check_file_licenses<P: AsRef<Path>>(path: P) {
 fn main() {
     // Check licenses in the current folder.
     check_file_licenses(".");
+
+    // Check if locktick feature is correctly enabled.
+    let locktick_enabled = env::var("CARGO_FEATURE_LOCKTICK").is_ok();
+    if locktick_enabled {
+        let profile = env::var("PROFILE").unwrap_or_else(|_| "".to_string());
```

Comment:
Is this unused at the moment?

Reply:
not sure what you mean?

Reply:
My bad, I didn't see where the variable profile was used, but I see it now